plat/layerscape/common/tsp/ls_tsp_setup.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <ls_def.h>
+#include <bl_common.h>
+#include <console.h>
+#include <platform_def.h>
+#include "platform_tsp.h"
+#include <plat_arm.h>
+#include <gicv2.h>
+#include <debug.h>
```

Comment:
nit: Leave a space after the list of includes.

Reply:
Done, thanks.